mini-project/BillManagementSystem/accountings/models.py

```diff
@@ -0,0 +1,40 @@
+from djmoney.models.fields import MoneyField
+from phonenumber_field.modelfields import PhoneNumberField
+
+from django.db import models
+from django.contrib.auth.models import User
+
+
+# Create your models here.
+class Payment(models.Model):
+    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    amount = MoneyField(max_digits=19, decimal_places=2, default_currency='MYR')
+    note = models.TextField()
+    created_at = models.DateTimeField()
+    updated_at = models.DateTimeField()
+    
+class Debt(models.Model):
+    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    amount = MoneyField(max_digits=19, decimal_places=2, default_currency='MYR')
+    created_at = models.DateTimeField()
+    updated_at = models.DateTimeField()
+    
+class Category(models.Model):
+    name = models.TextField()
```

Comment:
No unique constraints or of the sort? 😉

Reply:
I added max_length. Is that okay? 😀

Reply:
No problems with char length actually.
Just that imagine your users may potentially create 100 categories with name "ABC" 🤣

Reply:
Darn, I added a `unique=True` flag then, hope that it works. I can see this line of sql being generated now. 

```sql
CREATE TABLE "new__accounting_category" ("id" integer NOT NULL PRIMARY KEY AUTOINCREMENT, "name" text NOT NULL UNIQUE);
```

Reply:
Looks good